galley/pkg/runtime/source.go

```diff
@@ -88,8 +92,8 @@ func (s *InMemorySource) Stop() {
 	s.ch = nil
 }
 
-// Set the value in the in-memory store.
-func (s *InMemorySource) Set(k resource.Key, item proto.Message) {
+// Set the value in the in-memory store. Used for testing only.
```

Comment:
There is no other way to use in-memory store. :)
Used for testing only implies the method is used for testing, but actually the InMemoryStore is the one that is useful for testing.